ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/CCDCase.java

```diff
@@ -63,4 +68,16 @@
     private String features;
     private CCDYesNoOption migratedFromClaimStore;
     private String caseName;
+    private LocalDate docUploadDeadline;
+    private CCDDirectionPartyType docUploadForParty;
+    private LocalDate eyewitnessUploadDeadline;
+    private CCDDirectionPartyType eyewitnessUploadForParty;
+    private List<CCDOrderDirectionType> directionList;
+    private List<CCDOrderDirection> otherDirectionList;
```

Comment:
Above two can be renamed better.

Reply:
I've created https://tools.hmcts.net/jira/browse/ROC-5586 to clean up all of this. I currently don't have access to the template, and these fields are named in a funny way because that's what's specified in the template. 
Ideally I'd like to have code working, and then clean up the template / rename fields at the end